Content.Shared/Prying/Systems/PryingSystem.cs

```diff
@@ -109,7 +109,7 @@ private bool CanPry(EntityUid target, EntityUid user, out string? message, Pryin
 
         if (comp != null || Resolve(user, ref comp, false))
         {
-            canev = new BeforePryEvent(user, comp.PryPowered, comp.Force);
+            canev = new BeforePryEvent(user, comp.PryPowered, comp.Force, true);
```

Comment:
Can you check the paths to ensure that this is only tools?

This path's resolve looks like it's for mobs (e.g. xenos).

If existing functionally is fine, then maybe rename the bool to StrongPry instead of ToolUsed.